src/app/lib/config/services/bengali.js

```diff
@@ -67,7 +60,7 @@ export const service = {
       'https://www.bbc.com/bengali/institutional-50409861#authorexpertise',
     publishingPrinciples: 'https://www.bbc.com/bengali/institutional-50409861',
     isTrustProjectParticipant: true,
-    script,
+    script: bengaliNotoSerif,
```

Comment:
Can't you remove the use of script entirely?

Reply:
Nevermind, searched up what it does 🤦

Reply:
Can you expand? The script was originally set for the test environment so we've simply refactored it for live and it is needed in order to pull in the new font from gel foundations